packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -390,41 +400,76 @@ const Autocomplete = <
       (item: ReactElement & { children?: ReactElement[] }) =>
         [item].concat(item.children || []),
     );
-
-    // the height of an Odyssey autocomplete option item that is used to calculate height of window
-    const optionHeight = 45; //px
+    const sizeMap = useMemo(() => new Map<number, number>(), []);
```

Comment:
If we're only using this to store data, we'll want it as a `useRef` instead because those values are made to be mutated.